### ISSUE TITLE
Get all the loss log while training

### ISSUE THREAD

**Issue by kecyarchana** [jackfrost1411/MaskRCNN#5]
![loss](https://user-images.githubusercontent.com/22496671/147813429-4da84a87-3d7c-4294-a816-d0d35721ce52.png)

Hello, I am trying to implement MaskRCNN in my custom dataset (satellite image) to delineate trees.

How can I see all the loss like in the attachment during training? In my case, I only see loss and val_loss. Should I change anything in the compile function in model.py ?

Thank you for your time.
